Description: rpassword 7 doesn't appear to change the API compared to 6 in any way that matters for pleaser
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -45,7 +45,7 @@
 version = "1.7"
 
 [dependencies.rpassword]
-version = "6"
+version = ">= 6, < 8"
 
 [dependencies.syslog]
 version = ">= 6.0"
